Library/Expression/Closure.php

```diff
@@ -133,7 +152,36 @@ public function compile(array $expression, CompilationContext $compilationContex
 
         $symbolVariable->initVariant($compilationContext);
         $compilationContext->backend->createClosure($symbolVariable, $classDefinition, $compilationContext);
-
+        $compilationContext->headersManager->add('kernel/object');
+        foreach ($staticVariables as $var) {
+            if ('variable' == $var->getType() || 'array' == $var->getType()) {
+                $compilationContext->backend->updateStaticProperty($classDefinition->getClassEntry(), $var->getName(), $var, $compilationContext);
+            } else {
+                $tempVariable = $compilationContext->symbolTable->getTempNonTrackedVariable('variable', $compilationContext, true);
```

Comment:
@dreamsxin I think we'll use something like this:
```zep
fn () use (bar) {}
```
but not
```zep
fn () use (int bar) {}
```

By this, I think we don't need this branch. What do you think?

Reply:
zephir:
```zephir
		var abc = 1;
		return function() use (abc) {
			return abc + 1;
		};
```
c:
```c
	abc = 1;
	zephir_create_closure_ex(return_value, NULL, xxx_x__closure_ce, SL("__invoke"));
	ZEPHIR_INIT_ZVAL_NREF(tmp);
	ZVAL_LONG(&tmp, abc);
	zend_update_static_property(xxx_x__closure_ce, ZEND_STRL("abc"), &_0);
```